api/bucketsd/service.go

```diff
@@ -490,6 +490,10 @@ func (s *Service) pinBlocks(ctx context.Context, nodes []ipld.Node) (context.Con
 
 	// Check context owner's storage allowance
 	owner, ok := buckets.BucketOwnerFromContext(ctx)
+	if ok {
+		log.Debugf("pinBlock: storage: %d used, %d available, %d delta",
+			owner.StorageUsed, owner.StorageAvailable, owner.StorageDelta)
+	}
```

Comment:
Some logging around available storage that I'll use in a follow up PR addressing that issue.